.JuliaFormatter.toml

```diff
@@ -1,6 +1,14 @@
-margin = 120
+margin = 119
 always_for_in = true
-whitespace_in_kwargs = false
 remove_extra_newlines = true
 whitespace_ops_in_indices = true
-format_docstrings = true
\ No newline at end of file
+whitespace_in_kwargs = false
+format_docstrings = false
+separate_kwargs_with_semicolon = false
+verbose = true
+whitespace_typedefs = false
+join_lines_based_on_source = true
+surround_whereop_typeparameters = true
+trailing_comma = "nothing"
+align_conditional = true
+annotate_untyped_fields_with_any = true
```

Comment:
annotate_untyped_fields_with_any = true? I think this should be false.

trailing_comma = "nothing"? we could have true here I think.

Reply:
Yeah we can set untyped_fields to false.
For the trailing_comma, I set it to nothing because it seems we like using it in some cases and not in others, and I assumed there was a reason. But if there's no reason we can just enforce it. :)